contracts/Owned.sol

```diff
@@ -3,24 +3,24 @@ pragma solidity 0.5;
 contract Owned {
     address private owner;
 
-    event LogOwnerChanged(address indexed previousOwner, address newOwner);
+    event LogOwnerChanged(address indexed previousOwner, address indexed newOwner);
```

Comment:
This might be an anti-feature. Indexed values can't be read. They return hashes of "topics". This is great for searching and if you know the address you're concerned with, but not so great to listen for unexpected data from the data. 

Play around with it in Remix and you should see how it changes the character of the log.